pkgs/applications/networking/p2p/magnetico/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, fetchFromGitHub, buildGoModule, go-bindata }:
+
+buildGoModule rec {
+  pname = "magnetico";
+  version = "0.8.1";
+
+  src = fetchFromGitHub {
+    owner  = "boramalper";
+    repo   = "magnetico";
+    rev    = "v${version}";
+    sha256 = "1f7y3z9ql079ix6ycihkmd3z3da3sfiqw2fap31pbvvjs65sg644";
+  };
+
+  modSha256 = "1h9fij8mxlxfw7kxix00n10fkhkvmf8529fxbk1n30cxc1bs2szf";
+
+  buildInputs = [ go-bindata ];
+  buildPhase = ''
+    make magneticow magneticod
```

Comment:
I think you can use `buildFlags = [ "magneticow" "magneticod" ]` for this. Overriding `make` like this is discouraged because it prevents many customizations (e.g. no `-j` is passed)